### ISSUE TITLE
does not communicate any more

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#135]
```
What steps will reproduce the problem?
1. Install on phone, install on desktop
2. Excluded from norton!
3. Try test notification, anything


What is the expected output? What do you see instead?
It should produce something! It worked only once, yesterday.. it doesn't 
anymore. 
Oh yeah windows xp 32bit.

Please paste the application's log here (it should now be on your
clipboard):
Added as an attachment cause it was too long.

What version of Android are you using?
2.2 CM6 stable.. cyanogen's latest mod for N1


```

Original issue reported on code.google.com by `orestak...@gmail.com` on 13 Sep 2010 at 10:11

**Comment by GoogleCodeExporter** [comment]
```
Issue 136 has been merged into this issue.
```

Original comment by `rdamazio@gmail.com` on 13 Sep 2010 at 5:13

**Comment by GoogleCodeExporter** [comment]
```
Can you give more details on your setup - are you using bluetooth, wifi? If 
wifi, are the two devices on the same network?

Also adding the author of the desktop app.
```

Original comment by `rdamazio@gmail.com` on 13 Sep 2010 at 5:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdamazio@gmail.com` on 13 Sep 2010 at 5:16
- Changed state: **Waiting**

**Comment by GoogleCodeExporter** [comment]
```
Thank you very much for your support.

Problem was solved however, after I noticed that the balloon sound was there 
when I received anything on my phone, but the balloon was not, so my mobile 
communicated with the desktop notifier, it just didn't show the bubble on the 
notification area.

It turned out that something had de-activated my notification balloon service 
from the registry of Windows XP. I restored that and everything is now fine =)

Sincerely,
Orestis

P.S. Both of the devices were on the same wi-fi network, just for the record.
```

Original comment by `orestak...@gmail.com` on 13 Sep 2010 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
Great :)
```

Original comment by `rdamazio@gmail.com` on 14 Sep 2010 at 1:06
- Changed state: **Fixed**
